javascript/javascript1/week1/main.js

```diff
@@ -0,0 +1,44 @@
+// Age-ify (A future age calculator)
+
+const yearOfBirth = 1990; // number type variable
+let yearFuture = 2062; // number type variable
+let age = yearFuture - yearOfBirth;
+let futureAge = "You will be " + age + " years old in " + yearFuture;
+console.log(futureAge);
+
+// Goodboy-Oldboy (A dog age calculator)
+
+const dogYearOfBirth = 2010;
+let dogYearFuture = 2021;
+let dogYear = dogYearFuture - dogYearOfBirth;
+let shouldShowResultInDogYears = false; //boolean variable
+console.log(
+  "Your dog will be " + dogYear + " Human years old in " + dogYearFuture
+);
+shouldShowResultInDogYears = true;
+dogYear *= 7;
+console.log(
+  "Your dog will be " + dogYear + " dog years old in " + dogYearFuture
+);
```

Comment:
Technically you should have a conditional here, however we haven't gotten to them this module 😒 (Good exercises right there hyf). 

One minor thing to note about the variable names, especial `dogYear`, that is actually `humanYear` the first time around, so the description might be a bit off. 

But considering everything i will consider this task passed :)